psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -570,15 +570,22 @@ void DFHelper::prepare_AO_wK_core() {
     std::pair<size_t, size_t> plargest = pshell_blocks_for_AO_build(memory_, 1, psteps);
 
     wPpq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
-    Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
     m1Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
 
+    if (!wcombine_) Ppq_ = std::unique_ptr<double[]>(new double[big_skips_[nbf_]]);
+
+
     double* wppq = wPpq_.get();
-    double* ppq = Ppq_.get();
+    double* ppq;
```

Comment:
Set this to `nullptr`.  Setting to a known value will help if we ever have to debug this in the future.

Reply:
ok